analytics/tests.py

```diff
@@ -25,7 +25,7 @@
 
 API_URL = os.getenv("ANALYTICS_SERVER_URL", "http://localhost:8090")
 session_id = "00000000-0000-0000-0000-000000000000"
-result_id = "29cb352c-60c1-41d8-bfa1-7d6f7d955f63"
+result_id = "cdbd3bf6-1745-45bb-b399-61ee149cd58a"
```

Comment:
Why is this change necessary? Did we break something with the previous result id that we should be aware of for the future?

Reply:
As far as I know, the ID needs to match the identifier used in the API tests so that the stats exist. It's a result of all the integration tests being deeply dependent on each other #423.